guides/api.md

```diff
@@ -674,6 +664,32 @@ logger.info(ao.insertLogObject({
     more: 'there will be an added ao property'
 }))
 ```
+<a name="ao.wrapLambdaHandler"></a>
+
+### ao.wrapLambdaHandler([handler]) ⇒ <code>function</code>
+Wrap the lambda handler async function so it can be traced by AppOptics APM.
+
+**Kind**: static method of [<code>ao</code>](#ao)  
+**Returns**: <code>function</code> - - an async function, wrapping handler, to be used
+                      instead of handler.  
+
+| Param | Type | Description |
+| --- | --- | --- |
+| [handler] | <code>function</code> | wraps your lambda handler function so it                               is instrumented. your handler must return                               a promise if it is not an async function. |
```

Comment:
Maybe related to above comment, not sure if:
> your handler must return a promise if it is not an async function.
the "not async function" means the same thing as the "non-async handler" described in https://docs.aws.amazon.com/lambda/latest/dg/nodejs-handler.html... in which case do we only wrap if the customer's "non-async handler" returns a promise?

Reply:
that's actually impossible to know - what will the user's handler return?

we wrap the handler and allow it to complete via a promise or a callback. see `promiseOnlyHandler()`

Reply:
OK so why the `your handler must return a promise`?

Reply:
mostly because i wasn't focused on documenting what seems like a legacy, though still supported, interface. am changing to make it more clear.